### DISCUSSION TITLE
Segmentation fault (core dumped)

### DISCUSSION THREAD

**Discussion by NathanNguyen-Dev** [google/or-tools#3666] [Routing (and legacy CP) questions]
Hello, I am having a problem with a Segmentation fault (core dumped) for VRP application when I set the number of vehicles too high (around 200) If I set it to around 100 then it will usually work. Is there a way to avoid this? I based most of my work on this https://developers.google.com/optimization/routing/vrptw

**Comment by Mizux** [comment]
Do you have enough memory ?

**Comment by lperron** [comment]
I will move this to discussions. Most likely, this is a wrong model with bad indexes, bad time windows, node used in multiple pickup and deliveries...
